assets/scss/bulmaswatch/_variables.scss

```diff
@@ -0,0 +1,56 @@
+////////////////////////////////////////////////
+// PULSE
+////////////////////////////////////////////////
+$orange:       #FF7518;
```

Comment:
Color `#FF7518` should be written as `#ff7518`

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#hexnotation)